synfig-studio/src/gui/widgets/widget_timeslider.cpp

```diff
@@ -349,21 +388,65 @@ bool
 Widget_Timeslider::on_button_release_event(GdkEventButton *event){
 	SYNFIG_EXCEPTION_GUARD_BEGIN()
 	lastx = (double)event->x;
+	move_lower_bound_button = false;
+	move_upper_bound_button = false;
 	return event->button == 1 || event->button == 2;
 	SYNFIG_EXCEPTION_GUARD_END_BOOL(true)
 }
 
+void
+Widget_Timeslider::get_bounds_rectangle_dimensions(double& x0, double& x1, double& w ,bool lower)
```

Comment:
1. You don't need the width parameter.
When you use this method below, you only uses `x0` or `x1`.

2. space-comma issue before `lower`

Reply:
1- regarding one yes the width isn't necessary to be added but I actually made this function in order not to redo the same exact lines while getting the dimensions at various places so instead I just define the local variable and pass them to this function by reference to have them set up, kind of like a c style function. However, instead I can just manually define the width everywhere I need the coordinates. what do you think ?

2- oops didnt notice that one.